modules/byplayBidAdapter.js

```diff
@@ -0,0 +1,105 @@
+import * as utils from '../src/utils';
+import { config } from '../src/config';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { Renderer } from '../src/Renderer';
+import { VIDEO } from '../src/mediaTypes';
+
+const BIDDER_CODE = 'byplay';
+const ENDPOINT_URL = 'https://tasp0g98f2.execute-api.ap-northeast-1.amazonaws.com/v1/bidder';
+const VIDEO_PLAYER_URL = 'https://cdn.byplay.net/prebid-byplay-v2.js';
+const A1_CODE = '!function(e,r,t,a){if(!e.a1tracker){var n=e.a1tracker={};n.VERSION="v0.6.1",n.queue=n.queue||[];for(var s=0;s<t.length;s++){var u=t[s];n[u]=n[u]||function(e){return function(){for(var r=arguments.length,t=Array(r),a=0;a<r;a++)t[a]=arguments[a];n.queue.push({name:e,arguments:t.slice()})}}(u)}var c=r.createElement("script"),i=r.getElementsByTagName("script")[0];c.async=!0,c.src="//img.ak.impact-ad.jp/ut/a1/tracking.min.js",i.parentNode.insertBefore(c,i)}}(window,document,["ready","send","sendEvent","setParams","initParams","sendCustomerId","getSegments","getUserId"]);'
```

Comment:
This loads external JavaScript which is not permitted.

Please see: https://github.com/prebid/prebid-js-external-js-template#policy